### ISSUE TITLE
Cosmetic: request header description set as placeholder for related textbox

### ISSUE THREAD

**Issue by hlubovac** [swagger-api/swagger-ui#3920]
Don't get distracted by values, please. Pictures = 1000 words:

swagger-json; request params:
![image](https://user-images.githubusercontent.com/3127736/32913368-50616ad8-cacf-11e7-9c8c-866678ab9a04.png)

This is expected (`swagger-json > default` becomes `input.value`):
![image](https://user-images.githubusercontent.com/3127736/32913424-773e0b7a-cacf-11e7-9e50-11d111f1818a.png)

But, `input.placeholder` being set to `swagger-json > name + " - " + description` seems out of place:
![image](https://user-images.githubusercontent.com/3127736/32913388-5f1bfdea-cacf-11e7-859c-6621c3857ad2.png)

Rendered HTML:
![image](https://user-images.githubusercontent.com/3127736/32913448-8cf39660-cacf-11e7-8936-3f757186abf8.png)

How about not setting the `input.placeholder` to anything, if same information is displayed right next to it (name, description). There is nothing new to show, really. Maybe, it could be set to `default`, so once retyped and then cleared, the user can still see what it was (although it won't be much useful, since browsers remove the placeholder value the moment you start typing).

**Comment by hlubovac** [comment]
Also, textboxes for all request param seem to be placed within the "Description" column, while there's some space to the right. Not a big deal, but hurts aesthetics a little bit :)

![image](https://user-images.githubusercontent.com/3127736/32913859-e3fb777e-cad0-11e7-9322-731cc176156d.png)

**Comment by dalbani** [comment]
What about striping all markup from the placeholder value, using a library like https://github.com/remarkjs/strip-markdown?

**Comment by cmlccie** [comment]
I would like to provide (via some reasonable means) the text that Swagger UI will render as placeholder text.  Using the description, which is immediately above the text input field (redundant) and often doesn't fit inside the input control isn't attractive and doesn't add value to the UI.

**I suggest populating the placeholder text with either:**

1. **The `example` value.**  I don't like that the *example* values become *request defaults*.  If you don't want to make the example request, you have to delete the example values everytime you make a request using the UI.  Setting the example as the placeholder text seems more intuitive to me.  It shows the user an *example* of what could go in the field without providing the value as some sort of default.
2. **The `summary` value.**  If people really like having the example values prepopulated, that's okay.  One can just opt to not provide examples if they don't want this behavior.  In this case, using the `summary` text as the placeholder text would at least provide an option for providing more concise text (that might fit inside the text input control) and it wouldn't be redundant with the already provided `description`.

My $0.02.  😎

**Comment by Safrone** [comment]
Agreed that it would help usability to be able to specify the placeholder value explicitly since "`name` - `description`" is often redundant and hard to read

**Comment by GearTheWorld** [comment]
Any possible workaround right now to get only the name for the placeholder?
5.6.3
